docs/current/guides/406009-multiplatform-support.md

```diff
@@ -111,6 +111,18 @@ The only changes we made to enable faster cross-compilation are:
 
 The final image is still multi-platform because each `Container` set as a `PlatformVariant` was initialized with a specific platform (after the cross-compilation has occurred, at the bottom of the `for` loop in the code above).
 
+### Adding Caching Support
+
+Caching can significantly improve the speed of your builds in Dagger by saving and reusing the results of expensive operations, such as downloading dependencies or compiling code. Dagger supports caching via the `withMountedCache` function, which mounts a cache directory from the host machine into the container.
+
+We will compile the Go project twice. The first compilation will take longer because the cache directories are empty. The second compilation should be faster, as the cache directories will contain the necessary Go modules and build outputs from the first compilation:
```

Comment:
```suggestion
The following code listing compiles the Go project twice. The first compilation will take longer because the cache directories are empty. The second compilation should be faster, as the cache directories will contain the necessary Go modules and build outputs from the first compilation:
```